qiskit/algorithms/gradients/lin_comb_estimator_gradient.py

```diff
@@ -44,22 +50,34 @@ class LinCombEstimatorGradient(BaseEstimatorGradient):
     `arXiv:1811.11184 <https://arxiv.org/pdf/1811.11184.pdf>`_
     """
 
-    def __init__(self, estimator: BaseEstimator, **run_options):
+    def __init__(
+        self,
+        estimator: BaseEstimator,
+        derivative_type: DerivativeType = DerivativeType.REAL,
+        **run_options,
+    ):
         """
         Args:
             estimator: The estimator used to compute the gradients.
+            derivative_type: The type of derivative. Can be either ``DerivativeType.REAL``
+                ``DerivativeType.IMAG``, or ``DerivativeType.COMPLEX``. Defaults to
+                ``DerivativeType.REAL``.
+                For ``DerivativeType.REAL`` we compute 2Re[(dω⟨ψ(ω)|)O(θ)|ψ(ω)〉],
+                for ``DerivativeType.IMAG`` we compute 2Im[(dω⟨ψ(ω)|)O(θ)|ψ(ω)〉], and
+                for ``DerivativeType.COMPLEX`` we compute 2(dω⟨ψ(ω)|)O(θ)|ψ(ω)〉.
```

Comment:
The text here is formatted in way its easy to read as list items more or less. It all ends up, as it stands being wrapped as paragraph text so the result is not quite as nicely laid out

![image](https://user-images.githubusercontent.com/40241007/191601016-f0076130-9e96-42c5-bca8-b7dc1d961011.png)